.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "cSpell.words": ["dependancy"]
```

Comment:
Am I right in thinking this adds an exception for "dependancy"? I don't see where we use it in the code, do we need this here? Since it's misspelled

Reply:
That is a mistake 🙈 Will remove it. Good catch!

Reply:
Removed